modules/migration-configuring-mcg.adoc

```diff
@@ -11,30 +11,38 @@
+You must retrieve the Multicloud Object Gateway (MCG) credentials and S3 endpoint to configure MCG as a replication repository for the {mtc-full} ({mtc-short})
+
+You must retrieve the Multicloud Object Gateway (MCG) credentials to create a `Secret` custom resource (CR) for {mtc-short}.
+endif::[]
+
+ifdef::installing-oadp-mcg[]
+You must retrieve the Multicloud Object Gateway (MCG) credentials to create a `Secret` custom resource (CR) for the OpenShift API for Data Protection (OADP).
 endif::[]
-You must retrieve the Multicloud Object Gateway (MCG) credentials in order to create a `Secret` custom resource (CR) for the OpenShift API for Data Protection (OADP).
 //ifdef::installing-oadp-mcg[]
 //endif::[]
 
-MCG is a component of {rh-storage}.
+[NOTE]
+====
+Although the MCG Operator was link:https://catalog.redhat.com/software/containers/ocs4/mcg-rhel8-operator/5ddbcefbdd19c71643b56ce9?architecture=amd64&image=64243f5dcd0eb61355af9abd[deprecated], the MCG plugin is still available for {rh-storage}. To download the plugin, browse to link:https://access.redhat.com/downloads/content/547/ver=4/rhel---9/4.15.4/x86_64/product-software[Download {rh-storage-first}] and download the appropriate MCG plugin for your operating system.
+====
 
 .Prerequisites
 ifdef::openshift-origin[]
 * Ensure that you have downloaded the {cluster-manager-url-pull} as shown in _Obtaining the installation program_ in the installation documentation for your platform.
 +
 If you have the pull secret, add the `redhat-operators` catalog to the OperatorHub custom resource (CR) as shown in _Configuring {product-title} to use Red Hat Operators_.
 endif::[]
-* You must deploy {rh-storage} by using the appropriate link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9[OpenShift Data Foundation deployment guide].
+* You must deploy {rh-storage} by using the appropriate link:https://docs.redhat.com/en/documentation/red_hat_openshift_data_foundation/4.15[{rh-storage-first} deployment guide].
 
 .Procedure
-
-. Obtain the S3 endpoint, `AWS_ACCESS_KEY_ID`, and `AWS_SECRET_ACCESS_KEY` by running the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/managing_hybrid_and_multicloud_resources/accessing-the-multicloud-object-gateway-with-your-applications_rhodf#accessing-the-Multicloud-object-gateway-from-the-terminal_rhodf[`describe` command] on the `NooBaa` custom resource.
 ifdef::installing-3-4,installing-mtc[]
+* Obtain the S3 endpoint, `AWS_ACCESS_KEY_ID`, and `AWS_SECRET_ACCESS_KEY` by running the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.15/html/managing_hybrid_and_multicloud_resources/accessing-the-multicloud-object-gateway-with-your-applications_rhodf#accessing-the-Multicloud-object-gateway-from-the-terminal_rhodf[`describe` command] on the `NooBaa` custom resource.
```

Comment:
I reckon we should be updating links to access.redhat to the new docs.redhat.com 
```suggestion
* Obtain the S3 endpoint, `AWS_ACCESS_KEY_ID`, and `AWS_SECRET_ACCESS_KEY` by running the link:https://docs.redhat.com/en/documentation/red_hat_openshift_data_foundation/4.15/html/managing_hybrid_and_multicloud_resources/accessing-the-multicloud-object-gateway-with-your-applications_rhodf#accessing-the-Multicloud-object-gateway-from-the-terminal_rhodf`describe` command] on the `NooBaa` custom resource.
```